frontend/src/components/header/Header.tsx

```diff
@@ -64,14 +65,13 @@ const Header: React.FC<HeaderProps> = ({isLogged}: HeaderProps) => {
     }, [loggedUserType]);
 
     return (
-        <AppBar position="static" className="app-bar">
+        <AppBar className="app-bar">
             {isLogged && (
-                <Container maxWidth="xl">
+                <Container maxWidth="lg">
```

Comment:
if you can move it to css it would be good

Reply:
no, i can move it to styles.ts but why

Reply:
styles should be in css so that they do not interfere with reading the code